File: ce/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples31.vb
' <snippetlinqexamples31>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim firstcontact As Contact = svcContext.ContactSet _
                                .First()
  Console.WriteLine(firstcontact.LastName &amp; ", " _
                    &amp; firstcontact.FirstName _
                    &amp; " is the first contact")
  Console.WriteLine("==========================")
  Dim singlecontact As Contact =
      svcContext.ContactSet _
      .Single(Function(c) c.ContactId.Value.Equals(_contactId1))
  Console.WriteLine(singlecontact.LastName &amp; ", " _
                    &amp; singlecontact.FirstName _
                    &amp; " is the single contact")
 End Using

' </snippetlinqexamples31>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/linqexamples31.cs
// <snippetlinqexamples31>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  Contact firstcontact = svcContext.ContactSet.First();

  Contact singlecontact = svcContext.ContactSet.Single(c => c.ContactId == _contactId1);
  System.Console.WriteLine(firstcontact.LastName + ", " +
   firstcontact.FirstName + " is the first contact");
  System.Console.WriteLine("==========================");
  System.Console.WriteLine(singlecontact.LastName + ", " +
   singlecontact.FirstName + " is the single contact");
 }

// </snippetlinqexamples31>